MapboxNavigation/Resources/ca.lproj/Abbreviations.csv.plist

```diff
@@ -14,7 +14,7 @@
 &quot;Relleu del Litoral&quot;,&quot;lit.&quot;
 &quot;Municipi&quot;,&quot;mun.&quot;
 &quot;Orografia&quot;,&quot;orogr.&quot;
-&quot;Massís&quot;,&quot; Serra&quot;,&quot;serra&quot;
+&quot;Massís&quot;,&quot; Serra&quot;
```

Comment:
On second thought, “Serra” certainly isn’t an abbreviation for “Massís”. I think what happened was that the translator inadvertently omitted the abbreviation for “Massís” and put the “Serra” → “serra” item on the same line.

Reply:
Fixed in https://github.com/mapbox/mapbox-navigation-ios/pull/765